<launch>
  <node pkg="control_pkg" type="test_order_pub" name="test_order_pub">
  		<param name = "pubT" value = "0.1"/>
		<param name = "mode" value = "1"/>
		<param name = "time" value = "10"/>
		<param name = "R" value = "100"/>
		<param name = "L" value = "100"/>
		<param name = "yaw" value = "90"/>
		<param name = "dis" value = "1"/>
		<param name = "smooth" value = "false"/>
		
	</node>
</launch>
 
